packages/smart-wallet/src/smartWallet.js

```diff
@@ -439,10 +461,23 @@ export const prepareSmartWallet = (baggage, shared) => {
             },
             onStatusChange: offerStatus => {
               logger.info('offerStatus', offerStatus);
+
               updatePublishKit.publisher.publish({
                 updated: 'offerStatus',
                 status: offerStatus,
               });
+
+              const isSeatExited = 'numWantsSatisfied' in offerStatus;
```

Comment:
sort of odd to call this `isSeatExited` when the Zoe API includes a `hasExited()` API and we're not using it here.

Reply:
we don't have the seat here though. we could pass a "live" flag as part of the status but I think this is equivalent